script/update

```diff
@@ -299,6 +312,46 @@ def download(url, filename):
           break
         f.write(chunk)
 
+def get_packaged_src(clean, git_cache, nohooks, upload_src):
+  src_version = chromium_version()
+  tar_name = 'src.tar.bz2'
+  package_url = '{0}/{1}/src/{2}/{3}'.format(CHROMIUMCONTENT_SRC_URL,
+                                         PLATFORM_KEY, src_version, tar_name)
+  try:
+    print 'Trying to download ' + package_url
```

Comment:
wow it's been a while since i did python2 😅 would it make sense to use the method rather than the statement version of `print` everywhere? might make migrating to py3 easier when that day eventually comes.... any day now...